db/tx.go

```diff
@@ -9,10 +9,12 @@ import (
 	"github.com/BOPR/wallet"
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/ethereum/go-ethereum/common"
+	"github.com/jinzhu/gorm"
 	"github.com/kilic/bn254/bls"
 )
 
 var (
+	// ErrBadSignature error bad signature
```

Comment:
Can we avoid doc on self obvious code? The doc string does not provide helpful information.

Reply:
GoLang shows warnings :((

Reply:
I see, I'll look into how to disable that warning.